passthrough/delivery/http/passthrough_handler.go

```diff
@@ -61,6 +72,58 @@ func (a *PassthroughHandler) GetPortfolioAssetsByAddress(c echo.Context) error {
 	return c.JSON(http.StatusOK, portfolioAssetsResult)
 }
 
+func (a *PassthroughHandler) GetActiveOrdersStream(c echo.Context) (err error) {
+	ctx := c.Request().Context()
+
+	span := trace.SpanFromContext(ctx)
+	defer func() {
+		if err != nil {
+			span.RecordError(err)
+			// nolint:errcheck // ignore error
+			c.JSON(domain.GetStatusCode(err), domain.ResponseError{Message: err.Error()})
+		}
+
+		// Note: we do not end the span here as it is ended in the middleware.
+	}()
+
+	var req types.GetActiveOrdersRequest
+	if err := deliveryhttp.UnmarshalRequest(c, &req); err != nil {
+		return c.JSON(http.StatusBadRequest, domain.ResponseError{Message: err.Error()})
+	}
+
+	// Validate the request
+	if err := req.Validate(); err != nil {
+		return c.JSON(http.StatusBadRequest, domain.ResponseError{Message: err.Error()})
+	}
+
+	w := c.Response()
+	w.Header().Set("Content-Type", "text/event-stream")
+	w.Header().Set("Cache-Control", "no-cache")
+	w.Header().Set("Connection", "keep-alive")
+
+	ch := a.OUsecase.GetActiveOrdersStream(ctx, req.UserOsmoAddress)
+
+	for {
+		select {
+		case <-c.Request().Context().Done():
+			return c.NoContent(http.StatusOK)
+		case orders, ok := <-ch:
+			if !ok {
+				return c.NoContent(http.StatusOK)
+			}
+
+			if orders.Error != nil {
+				a.Logger.Error("GET "+c.Request().URL.String(), zap.Error(orders.Error))
```

Comment:
Do we want to respond with an error here @crnbarr93? 

It would look something like: "id: \ndata: { "message": "error" }\n\n"